vnext/src/IntegrationTests/AsyncStorageTest.windesktop.js

```diff
@@ -105,8 +123,9 @@ function testRemoveItem() {
       AsyncStorage.getAllKeys((err, result) => {
         expectAsyncNoError('testRemoveItem/getAllKeys', err);
         expectTrue(
-          result.indexOf(KEY_1) >= 0 && result.indexOf(KEY_2) >= 0,
-          'Missing KEY_1 or KEY_2 in ' + '(' + result + ')',
+          nullthrows(result).indexOf(KEY_1) >= 0 &&
+            nullthrows(result).indexOf(KEY_2) >= 0,
```

Comment:
>nullthrows [](start = 12, length = 10)

why to wrap up in nullthrows() every occurance of the result variable. It would throw in the first occurrence. BTW, without this function we would get an exception anyway: why is it needed in this context?

Reply:
See previous comments. Verbatim from RN original sources.